### ISSUE TITLE
make protocols detect completed iterations

### ISSUE THREAD

**Issue by azazellochg** [scipion-em/scipion-em-relion#33]
From https://github.com/I2PC/scipion/issues/547
maybe a way here could be to make the Relion protocols (and other that have the same iterative way and support continue from ) smarter and if they detect previous iterations, then will insert an step to continue from previous one....without the need to make a copy. Maybe this way complicates a little bit the parameters are or are not allowed in the continue mode.

**Comment by delarosatrevin** [comment]
Yes!!! I fully support this idea. I think is really weird for users that are used to Relion GUI, to make a copy and specify the continue. It should be more natural to make a "clever" continue as you suggest and just add a new step. 

I think this should not be that difficult, but I propose you to address this after we get rid of scipion-core 2.0.
